test/NuGet.Core.Tests/NuGet.Packaging.Test/PackageArchiveReaderTests.cs

```diff
@@ -1966,6 +1966,20 @@ public void CanVerifySignedPackages_ReturnsValueBasedOnOperatingSystemAndFramewo
             }
         }
 
+        [PlatformFact(Platform.Linux, Platform.Darwin)]
+        public void CanVerifySignedPackages_ReturnsValueBasedOnOperatingSystemAndFramework_Fails()
+        {
+            // Arrange
+            using (var test = TestPackagesCore.GetPackageContentReaderTestPackage())
+            using (var packageArchiveReader = new PackageArchiveReader(test))
+            {
+                // Act
+                var result = packageArchiveReader.CanVerifySignedPackages(null);
```

Comment:
Use the explicit type instead of `var` here and elsewhere in this PR per rule 10 in https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md.

Reply:
Ok. Addressed.